selfdrive/car/toyota/interface.py

```diff
@@ -147,6 +147,16 @@ def get_params(candidate, fingerprint):
       ret.mass = 4607 * CV.LB_TO_KG + std_cargo #mean between normal and hybrid limited
       ret.steerKpV, ret.steerKiV = [[0.6], [0.05]]
       ret.steerKf = 0.00006
+      
+    elif candidate == CAR.AVALON:
+      stop_and_go = False
+      ret.safetyParam = 100 # see conversion factor for STEER_TORQUE_EPS in dbc file
+      ret.wheelbase = 2.82
+      ret.steerRatio = 14.8 #Found at https://pressroom.toyota.com/releases/2016+avalon+product+specs.download
+      tire_stiffness_factor = 0.7983
+      ret.mass = 3505 * CV.LB_TO_KG + std_cargo  # mean between normal and hybrid
+      ret.steerKpV, ret.steerKiV = [[0.17], [0.03]]
```

Comment:
seems quite low. is it really better than standard 0.6/0.1?

Reply:
That’s what I though. It’s significantly better than 0.6/0.1